WalletWasabi/WabiSabi/Backend/Statistics/CoinJoinFeeRateStatStore.cs

```diff
@@ -0,0 +1,99 @@
+using NBitcoin;
+using NBitcoin.RPC;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using WalletWasabi.Bases;
+using WalletWasabi.BitcoinCore.Rpc;
+using WalletWasabi.WabiSabi.Models;
+
+namespace WalletWasabi.WabiSabi.Backend.Statistics;
+
+public class CoinJoinFeeRateStatStore : PeriodicRunner
+{
+	public CoinJoinFeeRateStatStore(WabiSabiConfig config, IRPCClient rpc, IEnumerable<CoinJoinFeeRateStat> feeRateStats)
+		: base(TimeSpan.FromMinutes(10))
+	{
+		Config = config;
+		Rpc = rpc;
+		CoinJoinFeeRateStats = new(feeRateStats.OrderBy(x => x.DateTimeOffset));
+	}
+
+	public CoinJoinFeeRateStatStore(WabiSabiConfig config, IRPCClient rpc)
```

Comment:
I would add a note `/// <remarks>Only for tests.</remarks>` or mark this `internal` so it's accessible only from tests.